temporalio/client.py

```diff
@@ -205,12 +285,14 @@ async def start_workflow(
         header: Optional[Mapping[str, Any]] = None,
         start_signal: Optional[str] = None,
         start_signal_args: Iterable[Any] = [],
-    ) -> WorkflowHandle[Any]:
+    ) -> WorkflowHandle[Any, Any]:
         """Start a workflow and return its handle.
 
         Args:
-            workflow: Name of the workflow to start.
-            args: Arguments for the workflow if any.
+            workflow: String name or class method decorated with
+                ``@workflow.run`` for the workflow to start.
+            arg: Single argument to the workflow.
+            args: Multiple arguments to the workflow.
```

Comment:
Maybe mention these are mutually exclusive?

Reply:
Will do (there are many places this applies)

Reply:
Done